dbsnp-importer/src/test/java/uk/ac/ebi/eva/dbsnpimporter/models/SubSnpCoreFieldsGetVariantKeyFieldsTest.java

```diff
@@ -0,0 +1,114 @@
+                                                         1, LocusType.SNP.intValue(), "12", 10144047L, 10144047L, "G",
+                                                         "G", null, null, "AC_000169.1:g.10144047delG", 10144047L,
+                                                         10144047L, 1, "AC_000169.1:g.10144047delG", 1591551L, 1591551L,
+                                                         1);
+
+        assertEquals(new VariantKeyFields("12", 10144047L, "G", ""), deletion.getVariantKeyFields());
+    }
+
+    @Test
+    public void multiNucleotideDeletionCoordinatesShouldNotChange() throws Exception {
+        SubSnpCoreFields deletion = new SubSnpCoreFields(1085240363L, 384020033L, 1, "NW_003103847.1", 1056819L,
+                                                         1056821L, 1, LocusType.DELETION.intValue(), "2", 100306584L,
+                                                         100306586L, "TCA", "TCA", null, null,
+                                                         "AC_000159.1:g.100306584_100306586delTCA", 100306584L,
+                                                         100306586L, 1, "NW_003103847.1:g.1056819_1056821delTCA",
+                                                         1056819L, 1056821L, -1);
+
+        assertEquals(new VariantKeyFields("2", 100306584L, "TCA", ""), deletion.getVariantKeyFields());
+    }
+
+    @Test
+    public void contigCoordinatesReturnedForDeletionNotMappedToChromosome() throws Exception {
+        SubSnpCoreFields deletion = new SubSnpCoreFields(1107437081L, 524371323L, 1, "NW_003101162.1", 229L, 232L, 1,
+                                                         LocusType.DELETION.intValue(), null, null, null, null, "TTTC",
+                                                         null, null, null, null, null, 1,
+                                                         "NW_003101162.1:g.229_232delTTTC", 229L, 232L, 1);
+
+        assertEquals(new VariantKeyFields("NW_003101162.1", 229L, "TTTC", ""), deletion.getVariantKeyFields());
+    }
+
+    @Test
+    public void singleNucleotideInsertionStartShouldBeAdjusted() throws Exception {
+        SubSnpCoreFields insertion = new SubSnpCoreFields(1092414490L, 522748169L, 1, "NW_003104285.1", 12118757L,
+                                                          12118758L, 1, LocusType.INSERTION.intValue(), "10",
+                                                          100013652L, 100013653L, "-", "-", "A", "-/A",
+                                                          "AC_000167.1:g.100013652_100013653insA", 100013652L,
+                                                          100013653L, 1, "NW_003104285.1:g.12118757_12118758insA",
+                                                          12118757L, 12118758L, 1);
+
+        assertEquals(new VariantKeyFields("10", 100013653L, "-", "A"), insertion.getVariantKeyFields());
```

Comment:
I think this test will fail, because in the function the start is not updated with a +1. maybe the function getVariantKeyFields can use getVariantCoordinates